virus/Lineari_obscurus.md

```diff
@@ -0,0 +1,106 @@
+
+Using the PebbleScout tool (Shiryev, S.A., Agarwala, R., 2024) to search against all metagenomic and metatranscriptomic runs released in the public SRA, however, a number of similar sequences to the *L. obscurus* genome were also found in BioSamples not listed in the Obelisk_cluster.csv file. While this document focuses analysis on the runs identified in the .csv file, further investigation leveraging the data from PebbleScout can prove to be insightful to the ecology of this obelisk.
+
+Though inferences regarding the hosts of *L. obscurus* remain limited, given that these obelisks are found in mice gut microbiomes, they may share similarities in their genome and genome products with those found in human gut microbiomes as characterized in Zheludev et al.
+
+To further investigate the sources of *L. obscurus*, I next sought to compile other BioSamples and run data within these BioProjects to explore the obelisk ecology found in these projects.
+
+**BioProjects containing *Lineari obscurus* revealed the presence of additional obelisks**
+
+In order to further investigate these BioProjects, I next compiled all individual runs recorded in the (file/obelisk_sraRunInfo.csv) for BioProjects A, B, and C. Each BioProject in Table 1. was found to be associated with a large number of BioSamples and individual runs. For example, ProjectA yielded 60 runs containing obelisks, a number far greater than the 4 runs containing *L. obscurus*.
+
+This observation then prompts the question: "Why is *L. obscurus* only present in a select few samples?". The additional obelisks found in these BioProjects and their relationships to *L. obscurus* remain to be investigated.
+
+In order to investigate the distribution of obelisks in these BioProjects, as well as their potential link to individual BioSamples, I further organized all data from each BioProject, focusing on the distribution of obelisks in each BioProject, the clusters they belong in, and their BioSample information.
+
+Interestingly, the collective data obtained from each BioProject contains a number of obelisks spread across multiple clusters, with notable overlaps in clusters 1813, 2131, and 2134 between the three projects. In addition, obelisks identified in run data from the same BioSample are sometimes found in different clusters - for example, obelisks from the BioSample SAMN06618381 are spread across clusters 674, 1812, 1813, 2131, and 2134; and obelisks from the BioSample SAMN12621060 are present in clusters 345, 2144, and 2145. This is rather unexpected because the material from the same BioSample is expected to yield the same results, though of course there may exist variations within the sample and technical factors that could affect consistency between runs.
+
+Table 1. Summary of SRA runs, BioSample, and BioProject associated with cluster 1812 obelisks.
+
+![Table 1](img/Lineari_obscurus/Table%201.png)
```

Comment:
for all images, please don't use whitspace in the filenames. Instead use an underscore.